base/acme/sbin/pki-acme-run

```diff
@@ -0,0 +1,184 @@
+#!/bin/sh
```

Comment:
do you want to stop the execution of the script if a particular command fails, by passing `-e`?

Reply:
Yes, I'll fix this too.

Reply:
Fixed.